resources/assets/less/app/kanban.less

```diff
@@ -54,6 +54,110 @@
   }
 }
 
+
+/* navpill */
+.kanban-nav {
+	position: relative;
+	margin: 0px 0;
+}
+.kanban-pill-tag {
+	color: #afafaf;
+	padding-left: 10px !important;
+	padding-right: 10px !important;
+}
+.kanban-nav ul {
+	margin: 0;
+	padding: 0;
+}
+.kanban-nav li {
+	margin: 0 5px 10px 0;
+	padding: 0;
+	list-style: none;
+	display: inline-block;
+}
+.kanban-nav a {
```

Comment:
To be honest I had my share of problems with this first pull request over more than just one file. First I tried the Git Desktop Client. I still have no idea how this thing's supposed to work. I guess for someone like me who is practically a GitHub virgin this it's not intended to be easy ;)  I still feel like being a complete idiot. In the end I tried GitKraken and made my commits with this one.

I made the pull request from within the Github site, after I pushed my local changes to Github.
But to be honest I have no idea how to split the changes into multiple pull requests.
Do I have to make different commits for this to be able to make different pull requests? 

Spaces/Tabs: Yeah. I'll try to make my editor use spaces instead of tabs right away.
This is something that bothers me a lot, but I haven't thought of this here.

Ok. Never heard of gulp before today. So it took me a while to get everything up and running and understand gulp and its functions.

What exactly does 'gulp production' do?